### DISCUSSION TITLE
Ridefinire state su card

### DISCUSSION THREAD

**Discussion by nclgius** [indomus/forum#581] [HUB: Home Assistant]
Ho configurato uno switch con MQTT sul file configure yaml:
`- platform: mqtt`
 ` unique_id: ESP_22_Tettoia_garage_switch_laterale`
   ` name: "Garage_laterale"`
   `state_topic: "ESP_22_Tettoia_garage/Serranda_laterale/State"`
`command_topic: "ESP_22_Tettoia_garage/Serranda_laterale/cmd"`
    `state_on: "1"`
    ` state_off: "0"`
    ` optimistic: false`
   `qos: 0`
   ` retain: false  `
    
Quando configuro la card e setto Show state, vorrei che comparisse aperto o chiuso invece di on off.
Grazie in anticipo

**Comment by snakuzzo** [comment]
Innanzi tutto correggi il post scrivendo correttamente il codice utilizzando il markdown (leggi il regolamento e le indicazioni [qui](https://github.com/indomus/forum/blob/main/README.md) ), altrimenti devo eliminare il topic.

Puoi provare ad usare le [device class](https://www.home-assistant.io/integrations/switch/#device-class), ma non credo che quelle previste per le entità di tipo switch tu possa cambiare in aperto/chiuso. Potresti invece usare una [template cover](https://www.home-assistant.io/integrations/cover.template/).

Guarda [questo topic](https://community.home-assistant.io/t/change-state-of-switch-from-on-off-to-open-closed/285837/4) per prendere spunto
